server/controllers/index.js

```diff
@@ -1,4 +1,7 @@
 const router = require('express').Router();
+const userRouter = require('./routes/user');
+
+router.use(userRouter);
```

Comment:
You should first declare variables and require modules then `router.use`

Reply:
i fix it :+1: